scrapper.py

```diff
@@ -2,6 +2,24 @@
 Crawler implementation
 """
 
+import json
```

Comment:
1. imports in alphabetical order
2. imports in following order: Python standard library, empty line, external libraries, empty line, local modules